homeassistant/components/eight_sleep/__init__.py

```diff
@@ -70,17 +53,29 @@ def _get_device_unique_id(eight: EightSleep, user_obj: EightUser | None = None)
 
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Set up the Eight Sleep component."""
+    """Old set up method for the Eight Sleep component."""
+    if DOMAIN in config:
```

Comment:
Suggestion: could save some indent if you reverse this condition and outdent below

Reply:
we'd also have to `return True` twice then